packages/botonic-plugin-contentful/src/contentful/export/locale-migrator.ts

```diff
@@ -0,0 +1,215 @@
+import { I18nFieldValues, SpaceExport } from './space-export'
+import assert from 'assert'
+// eslint-disable-next-line node/no-missing-import
+import { LocaleProps } from 'contentful-management/dist/typings/entities/locale'
+
+/**
+ * Useful to clone the contents flow from a space when the target locales are different.
+ */
+export class LocaleMigrator {
+  defaultLoc: string | undefined
+  /**
+   * @param fromLoc The locale whose flow we want to maintain
+   * @param toLoc The locale in which `fromLoc`'s flow will be converted
+   */
+  constructor(
+    readonly fromLoc: string,
+    readonly toLoc: string,
+    readonly verbose = false
+  ) {}
```

Comment:
The parameter assignment to class attributes is missing. How is this working?

Reply:
this is typescript! just having readonly, public or private does the declaration of the parameter, class attribute and assignment in one go :-)